config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'sessions#welcome'
```

Comment:
Is there a way to redirect to a certain page based on whether the user is logged in/out? 
Example:
A user loads up the page and you check that they are logged in and automatically send them to the posts page rather than having them click the view posts button and something similar for when they are logged out and send them to the log in or sign up page

Reply:
It should be doing that now! But if not we can hop on a call later to quickly change where the page is redirected to depending on the login status

Reply:
Yes, once the user logs in, it redirects straight to the posts page as it only has access if the user is logged in.